Sources/MapboxCoreNavigation/NavigationSettings.swift

```diff
@@ -149,6 +149,17 @@ public class NavigationSettings {
      */
     public static let shared: NavigationSettings = .init()
     
+    /// Returns a reflection of this class excluding the `properties` variable.
+    lazy var properties: [Mirror.Child] = {
```

Comment:
It was removed in main. Any reasons to bring this back?

Reply:
Was restored due to error rebase, fixed now.